CODE_REVIEW.md

```diff
@@ -0,0 +1,17 @@
+**TASK 1: Code Review**
+* Not all the Reading list actions have respective reducers.
+* Failure scenario for 'failedAddToReadingList' & 'failedRemoveFromReadingList' are expected to capture error, but 'item' is being passed. And their respective reducers are missing in reading-list.reducer.ts
```

Comment:
I think you've misunderstood what's going on here. We're doing optimistic updates in the UI, this means that the failure case has to undo what the initiating action does. This is why the failure actions have the `book` payload, so that it can subsequently be removed from/added to the list.

Reply:
Updated few more findings and made few changes as per optimistic UI updates in my recent commit.